docs/user-guide/node-api.md

```diff
@@ -4,20 +4,14 @@ The stylelint module includes a `lint()` function that provides the Node API.
 
 ```js
 stylelint.lint(options)
-  .then(function(resultObject) { .. });
+  .then(function(resultObject) { ... });
```

Comment:
Two periods is a convention I kind of like because it avoids confusion with spread/rest syntax in ES2015.